client/src/pages/CreateTournament.tsx

```diff
@@ -10,6 +9,7 @@ import Lightning from "../assets/logo/Lightning.svg";
 import ChooseMatchType from "../components/inputs/ChooseMatchType.tsx";
 import { MatchType } from "../../../types/socket/eventArguments.ts";
 import ButtonComponent from "../components/buttons/BorderedButtonComponent.tsx";
+import TournamentPopup from "../components/popups/TournamentPopup"; // Import the popup component
```

Comment:
Remove these inline comments. The only comments we should be making are "why" we are doing something, not "what" we are doing.